services/brig/test/integration/API/User/Connection.hs

```diff
@@ -62,7 +60,7 @@ tests ::
   FedClient 'Galley ->
   DB.ClientState ->
   TestTree
-tests cl _at opts p b _c g fedBrigClient fedGalleyClient db =
+tests cl _at p b _c g fedBrigClient _fedGalleyClient db =
```

Comment:
remove `_fedGalleyClient`?

Reply:
I thought of that, but then left it (in case someone needs it). Now that I think about it again, we won't ever update these tests, so no one will re-introduce the need for that federation client; hence I'll remove it.